### PR TITLE
fix: fix InternalDropdown search when label is not a string

### PR DESCRIPTION
**Pull request by HellWolf93**
## Changes proposed in this PR:
- Fix InternalDropdown search when item label is not a string

- [X] I have followed (at least) the [PR section of the contributing guide](https://github.com/nexxtway/react-rainbow/blob/master/CONTRIBUTING.md#submitting-a-pull-request).

### CODE REVIEW

**Review by LeandroTorresSicilia — APPROVED** [review]

File: src/components/Option/index.d.ts [hunk 1]
```diff
@@ -2,7 +2,8 @@ import { ComponentType, ReactNode } from 'react';
 import { BaseProps, IconPosition } from '../types';
 
 export interface OptionProps extends BaseProps {
-    label?: string;
+    label?: ReactNode;
+    labelText?: string;
```
Comment (TahimiLeonBravo) [inline src/components/Option/index.d.ts:6]: let's use "searchableText"

File: src/components/PicklistOption/index.d.ts [hunk 2]
```diff
@@ -2,7 +2,8 @@ import { ReactNode } from 'react';
 import { BaseProps, IconPosition } from '../types';
 
 export interface PicklistOptionProps extends BaseProps {
-    label?: string;
+    label?: ReactNode;
+    labelText?: string;
```
Comment (TahimiLeonBravo) [inline src/components/PicklistOption/index.d.ts:6]: ^^

**Review by TahimiLeonBravo — CHANGES_REQUESTED** [review]

File: src/components/PicklistOption/index.js [hunk 3]
```diff
@@ -13,7 +13,9 @@ export default function PicklistOption(props) {
 
 PicklistOption.propTypes = {
     /** Text of the PicklistOption. */
-    label: PropTypes.string,
+    label: PropTypes.node,
+    /** Searchable text when label is a node */
+    labelText: PropTypes.string,
```
Comment (TahimiLeonBravo) [inline src/components/PicklistOption/index.js:18]: ^^

File: src/components/PicklistOption/index.js [hunk 4]
```diff
@@ -39,6 +41,7 @@ PicklistOption.propTypes = {
 
 PicklistOption.defaultProps = {
     label: undefined,
+    labelText: undefined,
```
Comment (TahimiLeonBravo) [inline src/components/PicklistOption/index.js:44]: ^^

**Review by TahimiLeonBravo — CHANGES_REQUESTED** [review]

**Pull request merged**